base_export_manager/models/ir_exports_line.py

```diff
@@ -30,28 +11,119 @@ class IrExportsLine(models.Model):
     _inherit = 'ir.exports.line'
     _order = 'sequence,id'
 
+    name = fields.Char(
+        required=True,
+        readonly=True,
+        store=True,
+        compute="_compute_name",
+        inverse="_inverse_name",
+        help="Field's technical name.")
+    field1_id = fields.Many2one(
+        "ir.model.fields",
+        "First field",
+        domain="[('model_id', '=', model1_id)]")
+    field2_id = fields.Many2one(
+        "ir.model.fields",
+        "Second field",
+        domain="[('model_id', '=', model2_id)]")
+    field3_id = fields.Many2one(
+        "ir.model.fields",
+        "Third field",
+        domain="[('model_id', '=', model3_id)]")
+    model1_id = fields.Many2one(
+        string="First model",
+        readonly=True,
+        related="export_id.model_id")
```

Comment:
This field is not working in runbot. It is always false, no matter the value of `export_id.model_id`. I'd thank help on that too.